### DISCUSSION TITLE
rectangular selection ?

### DISCUSSION THREAD

**Discussion by mckellygit** [wezterm/wezterm#1186] [Q&A]
Hi,

Thank you for wezterm, I am learning the details.
Is there a rectangular selection mode available ?
I have not found anything about it yet.

-m

**Comment by wez** [comment]
sorry, wezterm doesn't have rectangular selection

**Comment by mckellygit** [comment]
Hi,
ok, thanks.  
This is one thing I use often, especially in copy-mode, so I will one day try to look into it and also hope it goes on the to-do list :-)
take care,
-m
